centaur/src/main/resources/standardTestCases/wdl_draft3/aws_local_disk/aws_local_disk.wdl

```diff
@@ -0,0 +1,25 @@
+version 1.0
+
+workflow test {
+
+  call local_disk {
+    input:
+      v = "latest"
+  }
+}
+
+task local_disk {
+
+  input {
+    String v
+  }
+
+  command {
+    echo "If we get here, then it worked!"
```

Comment:
Can we make this query the disk somehow (I'm thinking `df -kh` or similar) and then assert correctness in the `.test` file?